test/detached.js

```diff
@@ -27,7 +27,7 @@ describe('detached DOM node', function() {
       if (typeof MutationObserver === 'function') {
         describe('when the browser supports `MutationObserver`', function () {
 
-          beforeEach(h.wait(50));
```

Comment:
By increasing the waiting time, I've got this test always pass on local and saucelabs.